src/main/java/reposense/authorship/FileInfoAnalyzer.java

```diff
@@ -122,7 +122,8 @@ private static FileResult generateBinaryFileResult(RepoConfiguration config, Fil
         for (String authorString : authorsString.split("\n")) {
             String[] arr = authorString.split("\t");
             String authorName = arr[0];
-            String authorEmail = arr[1];
+            String authorEmail = arr.length == 1 ? "" : arr[1];
```

Comment:
The best place to put this might be in the `GitLog::getBinaryFileAuthors` method itself. However, I believe most of the `git` commands especially those in `git log` return the raw output from the command without any parsing though there are exceptions like those in `GitRevList`. 

However, I think it is a good point. The git log command being run here uses a very specific format that the user of this method must take into consideration and parse accordingly (tab-separated name and email).  I'll use a length 2 String array instead of a Pair since there isn't a Pair class.

Reply:
I've updated accordingly.